pandas/tests/strings/test_split_partition.py

```diff
@@ -92,81 +103,42 @@ def test_split_object_mixed():
     assert isinstance(result, Series)
     tm.assert_almost_equal(result, exp)
 
-    result = mixed.str.split("_", expand=False)
-    assert isinstance(result, Series)
-    tm.assert_almost_equal(result, exp)
-
 
 @pytest.mark.parametrize("method", ["split", "rsplit"])
-def test_split_n(any_string_dtype, method):
+@pytest.mark.parametrize("n", [None, 0])
+def test_split_n(any_string_dtype, method, n):
     s = Series(["a b", pd.NA, "b c"], dtype=any_string_dtype)
     expected = Series([["a", "b"], pd.NA, ["b", "c"]])
 
-    result = getattr(s.str, method)(" ", n=None)
-    tm.assert_series_equal(result, expected)
-
-    result = getattr(s.str, method)(" ", n=0)
+    result = getattr(s.str, method)(" ", n=n)
     tm.assert_series_equal(result, expected)
 
 
 def test_rsplit(any_string_dtype):
-    values = Series(["a_b_c", "c_d_e", np.nan, "f_g_h"], dtype=any_string_dtype)
```

Comment:
don't you need these cases?

Reply:
Parameterized now in `test_split` and  `test_split_more_than_one_char`,